### DISCUSSION TITLE
Could you please tell what was the last Windows 32 supported version?

### DISCUSSION THREAD

**Discussion by rin67630** [debauchee/barrier#1266] [Q&A]
Many legacy systems / tablets are running 32 bit.
Synergy ran on them.
Could you please at least mention which was your last 32 bit compatible version?
Thank you.

**Comment by AdrianKoshka** [comment]
To my knowledge, you should still be able to build barrier for 32-bit windows systems, we just don't officially support 32-bit versions of windows.

**Comment by rin67630** [comment]
The installer refuses to install on a 32 bit tablet.

**Comment by AdrianKoshka** [comment]
We only support 64-bit windows for the installer.

**Comment by mirh** [comment]
The installer still seems to install just fine even on x86 tbh. 
You are only greeted with the hard reality once you try to run the program.

**Comment by shymega** [comment]
Yes, its true that the installer works fine on i386. My personal opinion is that we should begin supporting 32-bit again.. but I know there's a bit of opposition there. @debauchee/barrier-developers - thoughts? Reopening for further discussion.

**Comment by p12tic** [comment]
I think that support as many architectures as reasonably possible should be our goal.

**Reply by Shreeniwas1** [reply]
hello can you guys make it for 32 bit i wanted it on my old pc if possibe please

**Reply by shymega** [reply]
I and @p12tic forked Barrier to our own fork due to organisational issues, but we are still working to make a release. Sorry, but we're no longer active on the Barrier project.
